deploy/gatekeeper.yaml

```diff
@@ -789,7 +789,7 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: metadata.name
-        image: openpolicyagent/gatekeeper:v3.5.0-rc.1
+        image: openpolicyagent/gatekeeper:v3.5.0
```

Comment:
can you pls also bump https://github.com/open-policy-agent/gatekeeper/blob/master/deploy/experimental/gatekeeper-mutation.yaml

Reply:
yea as a seperate pr

Reply:
added to this so tag matches